.gitignore

```diff
@@ -49,3 +49,35 @@ hs_err_pid*
 # data which is too big to put into git
 data/On_Time_On_Time*
 data/2016_*.csv
+data/readme.html
```

Comment:
It's a description of the dataset. It's included in the .zip that is downloaded by `data/download.sh`.